libraries/apollo-api/api/apollo-api.klib.api

```diff
@@ -8,69 +8,63 @@
-abstract class com.apollographql.apollo.api/ObjectMap : kotlin.collections/Map<kotlin/String, kotlin/Any?> { // com.apollographql.apollo.api/ObjectMap|null[0]
-    constructor <init>(kotlin.collections/Map<kotlin/String, kotlin/Any?>) // com.apollographql.apollo.api/ObjectMap.<init>|<init>(kotlin.collections.Map<kotlin.String,kotlin.Any?>){}[0]
-    open fun containsKey(kotlin/String): kotlin/Boolean // com.apollographql.apollo.api/ObjectMap.containsKey|containsKey(kotlin.String){}[0]
-    open fun containsValue(kotlin/Any?): kotlin/Boolean // com.apollographql.apollo.api/ObjectMap.containsValue|containsValue(kotlin.Any?){}[0]
-    open fun get(kotlin/String): kotlin/Any? // com.apollographql.apollo.api/ObjectMap.get|get(kotlin.String){}[0]
-    open fun isEmpty(): kotlin/Boolean // com.apollographql.apollo.api/ObjectMap.isEmpty|isEmpty(){}[0]
-    open val entries // com.apollographql.apollo.api/ObjectMap.entries|{}entries[0]
-        open fun <get-entries>(): kotlin.collections/Set<kotlin.collections/Map.Entry<kotlin/String, kotlin/Any?>> // com.apollographql.apollo.api/ObjectMap.entries.<get-entries>|<get-entries>(){}[0]
-    open val keys // com.apollographql.apollo.api/ObjectMap.keys|{}keys[0]
-        open fun <get-keys>(): kotlin.collections/Set<kotlin/String> // com.apollographql.apollo.api/ObjectMap.keys.<get-keys>|<get-keys>(){}[0]
-    open val size // com.apollographql.apollo.api/ObjectMap.size|{}size[0]
-        open fun <get-size>(): kotlin/Int // com.apollographql.apollo.api/ObjectMap.size.<get-size>|<get-size>(){}[0]
-    open val values // com.apollographql.apollo.api/ObjectMap.values|{}values[0]
-        open fun <get-values>(): kotlin.collections/Collection<kotlin/Any?> // com.apollographql.apollo.api/ObjectMap.values.<get-values>|<get-values>(){}[0]
-    // Targets: [js]
-    open fun asJsReadonlyMapView(): kotlin.js.collections/JsReadonlyMap<kotlin/String, kotlin/Any?> // com.apollographql.apollo.api/ObjectMap.asJsReadonlyMapView|asJsReadonlyMapView(){}[0]
+final enum class com.apollographql.apollo.api.http/HttpMethod : kotlin/Enum<com.apollographql.apollo.api.http/HttpMethod> { // com.apollographql.apollo.api.http/HttpMethod|null[0]
+    enum entry Get // com.apollographql.apollo.api.http/HttpMethod.Get|null[0]
+    enum entry Post // com.apollographql.apollo.api.http/HttpMethod.Post|null[0]
+
+    final val entries // com.apollographql.apollo.api.http/HttpMethod.entries|#static{}entries[0]
+        final fun <get-entries>(): kotlin.enums/EnumEntries<com.apollographql.apollo.api.http/HttpMethod> // com.apollographql.apollo.api.http/HttpMethod.entries.<get-entries>|<get-entries>#static(){}[0]
+
+    final fun valueOf(kotlin/String): com.apollographql.apollo.api.http/HttpMethod // com.apollographql.apollo.api.http/HttpMethod.valueOf|valueOf#static(kotlin.String){}[0]
+    final fun values(): kotlin/Array<com.apollographql.apollo.api.http/HttpMethod> // com.apollographql.apollo.api.http/HttpMethod.values|values#static(){}[0]
```

Comment:
The Klib API dump changed **a lot**. Not reporting upstream because Klib support is still experimental and I guess this is to be expected.